www-cronrobot/app/javascript/stylesheets/main-dashboard.scss

```diff
@@ -1,28 +1,233 @@
+$bg-color: #222121;
```

Comment:
- fields are not properly grouped. The name should be alone on the first line

- then you should have 2 grouped fields with a nice border and should be named Schedule: Custom, and schedule. both are in fact a single field, and it should be displayed so that the user knows about it.

- The remote command should be separated

Then there should be a last section "Options", which can be toggle (visible, not visible).
When it is toggled, you should see for now Notification Channels, where he can pick one or many notification channels.

![image](https://user-images.githubusercontent.com/8007722/132255281-b416c26c-915f-48f5-be27-2f26e1e84691.png)

Reply:
following your changes, some other comments:

- if the options could be switched off
- if the scheduler could be more clear they form a group of fields:
![image](https://user-images.githubusercontent.com/8007722/132397448-1ff69e1e-4b46-4d8d-848a-f739cde5b31b.png)